tests/gnoi_reboot_test.py

```diff
@@ -23,13 +23,14 @@
 
 REBOOTMETHOD_UNKNOWN_ENUM = 0
 REBOOTMETHOD_COLD_BOOT_ENUM = 1
-REBOOTMETHOD_NSF_ENUM = 5
+REBOOTMETHOD_WARM_BOOT_ENUM = 4
+#REBOOTMETHOD_NSF_ENUM = 6 
```

Comment:
Do these enums have a dependency on some other values ? Why is 5 changed to 4 ? And can we remove the commented line ?

Reply:
@kishanps, In gnoi (https://github.com/openconfig/gnoi/blob/main/system/system.proto), Warm reboot is set to 4.  So, I have set to 4.